### DISCUSSION TITLE
YOLOv11 with torch hub?

### DISCUSSION THREAD

**Discussion by DylanBPY** [ultralytics/ultralytics#16976] [Q&A] [answered]
When will torch hub support be available for YOLOv11? I run all of my models via torch.hub.load and it would be nice to use a model through that. Also, where is the location of the train file, (like yolov5 and yolov8 have)?

Thanks
- Dylan

**Comment by glenn-jocher** [comment]
@DylDevs torch Hub support for YOLOv11 is not available yet, but you can follow updates on our GitHub. The train file is located in the `ultralytics` directory. For more details, visit our documentation at https://docs.ultralytics.com/hub/.

**Reply by DylanBPY** [reply]
Has any progress been made on support for torch hub?

**Reply by glenn-jocher** [reply, marked as answer]
Thank you for your interest. Currently, there is no support for YOLOv11 on Torch Hub, but please stay tuned for updates on our GitHub repository.
